paddle/fluid/operators/jit/helper.cc

```diff
@@ -109,12 +112,18 @@ void pack_weights<float>(const float* src, float* dst, int n, int k) {
   int block, rest;
   const auto groups = packed_groups(n, k, &block, &rest);
   std::for_each(groups.begin(), groups.end(), [&](int i) {
-    PADDLE_ENFORCE_GT(i, 0, "each element of groups should be larger than 0.");
+    PADDLE_ENFORCE_GT(i, 0, platform::errors::InvalidArgument(
+                                "Each element of groups should be larger than "
```

Comment:
here I don't understand, if `i < 0`, such as -1, `the -1th element`?

Reply:
replace with the accurate value